src/js/vre/chapter31/config/form.js

```diff
@@ -159,12 +166,62 @@ const formConfig = {
     contactInformation: {
       title: 'Contact Information',
       pages: {
+        veteranAddress: {
+          path: 'veteran-address',
+          title: 'Address Information',
+          uiSchema: {
+            veteranAddress: address.uiSchema(''),
+            'view:isMoving': {
+              'ui:title': 'Are you moving within the next 30 days?',
+              'ui:widget': 'yesNo'
+            },
+            veteranNewAddress: _.merge(
+              address.uiSchema('New address', false, (formData) => formData['view:isMoving']),
+              {
+                'ui:options': {
+                  expandUnder: 'view:isMoving'
+                }
+              }
+            )
+          },
+          schema: {
+            type: 'object',
+            required: ['veteranAddress', 'view:isMoving'],
+            properties: {
+              veteranAddress: address.schema(fullSchema31, true),
+              'view:isMoving': {
+                type: 'boolean'
+              },
+              veteranNewAddress: address.schema(fullSchema31)
+            }
+          }
+        },
         contactInformation: {
           path: 'contact-information',
           title: 'Contact Information',
+          uiSchema: {
+            daytimePhone: phoneUI('Daytime phone number'),
+            eveningPhone: phoneUI('Evening phone number'),
+            email: {
+              'ui:title': 'Email address'
+            },
+            'view:confirmEmail': {
+              'ui:title': 'Re-enter email address',
+              'ui:options': {
+                hideOnReview: true
```

Comment:
Should I be adding this to other booleans like `view:isMoving`

Reply:
I think you can leave it off of isMoving. We mainly use this for data that isn't useful to review.